packages/google-compute-engine-oslogin/google_oslogin_control

```diff
@@ -347,18 +361,34 @@ remove_google_dirs() {
 }
 
 activate() {
-  for func in modify_sshd_conf modify_nsswitch_conf \
-              modify_pam_config setup_google_dirs restart_svcs restart_sshd \
-              modify_group_conf; do
+  if is_freebsd; then
```

Comment:
the only difference here is one function.. i think that function can just short-circuit in freebsd case, instead.

Reply:
Right, this is a better solution.